tools/tertiary-analysis/pyscenic/pyscenic_ctx.xml

```diff
@@ -0,0 +1,142 @@
+<tool id="pyscenic_ctx" name="PySCENIC CTX" profile="21.09" version="0.12.1+galaxy0">
+    <description>
+        computes active regulons based on a gene regulatory network
+    </description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements"/>
+    <command><![CDATA[
+    PySCENIC_DB=db.genes_vs_motifs.rankings.feather &&
```

Comment:
```suggestion
    #set PySCENIC_DB = db.genes_vs_motifs.rankings.feather
```

Would make the commandline simpler and not expose and ENV var. Maybe you find that simpler.

Reply:
Thanks!